### ISSUE TITLE
[packager] `registry.DEFAULT.password must not be blank` while not using the docker registry

### ISSUE THREAD

**Issue by gotson** [jreleaser/jreleaser#1232]
I get the following when running `gradle jreleaserFullRelease`

```
[ERROR] == JReleaser ==
[ERROR] registry.DEFAULT.password must not be blank. Configure a value using the Gradle DSL, or define [jreleaser.distribution.komga.docker.default.password, jreleaser.docker.default.password] System properties, or define [JRELEASER_DISTRIBUTION_KOMGA_DOCKER_DEFAULT_PASSWORD, JRELEASER_DOCKER_DEFAULT_PASSWORD] environment variables, or define a key/value pair in /home/runner/.jreleaser/config.properties with keys [JRELEASER_DISTRIBUTION_KOMGA_DOCKER_DEFAULT_PASSWORD, JRELEASER_DOCKER_DEFAULT_PASSWORD]
```

Here is my relevant configuration for docker:

```kotlin
  packagers {
    docker {
      active.set(Active.RELEASE)
      templateDirectory.set(projectDir.resolve("docker"))
      repository.active.set(Active.NEVER)
      buildArgs.set(listOf("--cache-from", "gotson/komga:latest"))
      imageNames.set(
        listOf(
          "komga:latest",
          "komga:{{projectVersion}}",
          "komga:{{projectVersionMajor}}.x",
        ),
      )
      registries {
        create("docker.io") { externalLogin.set(true) }
        create("ghcr.io") { externalLogin.set(true) }
      }
      buildx {
        enabled.set(true)
        createBuilder.set(false)
        platforms.set(
          listOf(
            "linux/amd64",
            "linux/arm/v7",
            "linux/arm64/v8",
          ),
        )
      }
    }
  }
```

Given `repository.active.set(Active.NEVER)`, i don't understand why the issue is raised.

### Task List

- [ ] Steps to reproduce provided
- [ ] Stacktrace (if present) provided
- [ ] Example that reproduces the problem (link to git repository is ideal)
- [ ] Full description of the issue provided (see below)

### Steps to Reproduce

1. TODO
2. TODO
3. TODO

### Expected Behaviour

Should run fine

### Actual Behaviour

Errors out

### Environment Information

- **Operating System**: ubuntu
- **JReleaser Version:** 1.5.0
- **JDK Version:** 8

Please paste stacktraces from `[out|build|target]/jreleaser/trace.log`.

**Comment by aalmiray** [comment]
Workaround is to use `DEFAULT` instead of 'docker.io`.

**Comment by gotson** [comment]
My workaround was to add `JRELEASER_DOCKER_DEFAULT_PASSWORD: workaround` in the Github workflow file for that step!

**Comment by gotson** [comment]
It might actually be linked to https://github.com/jreleaser/jreleaser/discussions/1233 if the repository is considered active

**Comment by aalmiray** [comment]
Perhaps. Registries and repositories are independent from one another but this could be a side effect of resolving `active` via env vars. good catch.

**Comment by gotson** [comment]
I just tested and this is not related. Even if the repository is disabled via environment variable, this error still pops up.

**Comment by aalmiray** [comment]
The bug is caused by duplicate entries of the `DEFAULT` registry, one of which has `externalLogin` set to `false` as default.

**Comment by gotson** [comment]
> Workaround is to use `DEFAULT` instead of 'docker.io`.

Tried this but didn't work, same error.

**Comment by aalmiray** [comment]
Indeed, because there's still an internal registry named `DEFAULT` which you can't see nor update. 

Line 361 should be after 374 

https://github.com/jreleaser/jreleaser/blob/36b04bf3818c6233288b29626601f323ed0ae461/core/jreleaser-model-impl/src/main/java/org/jreleaser/model/internal/validation/packagers/DockerPackagerValidator.java#L361

**Comment by aalmiray** [comment]
🎉 This issue has been resolved in `v1.5.1` ([Release Notes](https://github.com/jreleaser/jreleaser/releases/tag/v1.5.1))
